avocado/formatters.py

```diff
@@ -95,139 +105,238 @@ def to_FORMAT(value, field=field, concept=self.concept,
         def to_FORMAT(values, fields=self.fields, concept=self.concept,
                       process_multiple=True, **context)
 
-    `values` will be an OrderedDict of values for each field. `fields` is
+    `values` will be a record of ordred values for each field. `fields` is
     a map of `DataField` instances associated with `concept` keyed by their
     natural key starting with `field_name` and prepending the `model_name`
     and `app_name` to prevent key collisions.
 
-    The output of a `process_multiple` method can be a single formatted value,
-    an OrderedDict, or a sequence of key-value pairs.
+    The output of a `process_multiple` method must be a tuple of values.
     """
-    def __init__(self, concept=None, keys=None):
+    def __init__(self, concept=None, keys=None, formats=None):
```

Comment:
Would it be better to set `formats=[]` as the default to avoid conditional checking of formats in some spots below?